sklearn/_min_dependencies.py

```diff
@@ -25,6 +25,7 @@
     "threadpoolctl": (THREADPOOLCTL_MIN_VERSION, "install"),
     "cython": (CYTHON_MIN_VERSION, "build"),
     "meson-python": ("0.16.0", "build"),
+    "ninja": ("1.8.2", "build"),
```

Comment:
Is the minimum version of Ninja documented somewhere? I guess the minimum version of Ninja might depend on the minimum version of Meson...

Current versions of Meson depend on Ninja 1.8.2 or newer. From the [README](https://github.com/mesonbuild/meson/blob/master/README.md):

> #### Dependencies
> * [Python](https://python.org/) (version 3.7 or newer)
> * [Ninja](https://ninja-build.org/) (version 1.8.2 or newer)